themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,148 @@
+
+## Deploying the project
+
+The template requires no additional configuration. Once the new project is created, you can deploy it immediately with [`pulumi up`]({{< relref "/docs/reference/cli/pulumi_up" >}}):
+
+```bash
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) values:
+
+originURL
+: The HTTP URL of the static website.
+
+apiURL
+: The HTTP URL of the serverless function endpoint.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources. The computed `originURL`, for example, can be used from the command line to open the newly deployed website in your favorite web browser:
+
+```bash
+$ open $(pulumi stack output originURL)
+```
+
+## Customizing the project
+
+Projects created with the serverless template expose the following [configuration]({{< relref "/docs/intro/concepts/config" >}}) settings:
+
+www_path
+: The path to the folder containing the files of the website. Defaults to `www`, which is the name (and relative path) of the folder included with the template.
+
+api_path
+: The path to the folder containing the files of the serverless function. Defaults to `api`, which is the name (and relative path) of the folder included with the template.
+
+index_document
+: The file to use for top-level pages. Defaults to `index.html`.
+
+error_document
+: The file to use for error pages. Defaults to `error.html`.
+
+All of these settings are optional and may be adjusted either by editing the stack configuration file directly (by default, `Pulumi.dev.yaml`) or by changing their values with [`pulumi config set`]({{< relref "/docs/reference/cli/pulumi_config_set" >}}) as shown below.
+
+### Using your own web content
+
+If you already have a serverless application you'd like to deploy on Azure with Pulumi, you can do so either by replacing placeholder content in the `www` folder or by configuring the stack to point to another folder on your computer with the `www_path` setting:
+
+```bash
+$ pulumi config set www_path ../my-existing-website/build
+$ pulumi up
+```
+
+## Tidying up
+
+You can cleanly destroy the stack and all of its infrastructure with [`pulumi destroy`]({{< relref "/docs/reference/cli/pulumi_destroy" >}}):
+
+```bash
+$ pulumi destroy
+```
+
+## Learn more
+
+Congratulations! You're now well on your way to managing a production-grade serverless application on Azure with Pulumi --- and there's lots more you can do from here:
+
+* Discover more architecture templates in [Templates &rarr;]({{< relref "/templates" >}})
+* Dive into the Azure Native package by exploring the [API docs in the Registry &rarr;]({{< relref "/registry/packages/azure-native" >}})
+* Expand your understanding of how Pulumi works in [Pulumi Learn &rarr;]({{< relref "/learn" >}})
+* Read up on the latest new features [in the Pulumi Blog &rarr;](/blog/tag/serverless)
```

Comment:
should this go to serverless or azure? what do we think people are going to be interested in? @cnunciato

Reply:
Probably Azure, since it's more likely to spark interest. (If you're an Azure person, you'll probably be more interested in seeing more of what you can do with Azure than what Pulumi can do that qualifies as "serverless".)

```suggestion
* Read up on the latest new Azure features [in the Pulumi Blog &rarr;](/blog/tag/azure)
```